mimic/test/test_neutron.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests for neutron api
+"""
+
+from __future__ import absolute_import, division, unicode_literals
+from twisted.trial.unittest import SynchronousTestCase
+from mimic.test.helpers import json_request
+from mimic.rest.neutron_api import NeutronApi
+from mimic.test.fixtures import APIMockHelper
+
+
+class NeutronTests(SynchronousTestCase):
+    """
+    Tests for networks using the Neutron Api plugin.
+    """
+    def setUp(self):
+        """
+
```

Comment:
On second thought this is a pretty minor nitpick, so just do it in a follow-up PR…